### ISSUE TITLE
num_workers for FastTensorDataLoader

### ISSUE THREAD

**Issue by tomanick** [hcarlens/pytorch-tabular#1]
Thanks for your sharing! Nice Work!
However, I have a question about num workers for loading dataset.
In pytorch dataloader, there is a num worker mechanism for multi-thread loading data.
How to set num workers in the FastTensorDataLoader or is there similar mechanism in FastTensorDataLoader?

**Comment by hcarlens** [comment]
Thanks! 
It's been a while since I looked at this code, but it looks like I didn't implement an ability to have multiple workers here. 
I do remember having issues with the `num_workers` parameter in the standard dataloaders when I first tried using that under Windows, which made it slower (though I do think it works well on macOS/Linux). 
I also feel like having multiple workers might not really help in this specific case, since it's loading data that's already in memory and not really doing anything to it - but I'm happy to be proven wrong!
There are actually some additional optimisation that can be made to this dataloader to make it much faster still - maybe I'll get around to those sometime. 

Harald
